advisories/github-reviewed/2021/05/GHSA-hgwp-4vp4-qmm2/GHSA-hgwp-4vp4-qmm2.json

```diff
@@ -28,7 +28,7 @@
               "introduced": "0"
             },
             {
-              "fixed": "2020.8.1"
+              "fixed": "0.0.0-20200820025921-9323844ea773"
```

Comment:
This version was generated from the tag "2020.8.1".

https://github.com/cloudflare/cloudflared/releases/tag/2020.8.1